Client/Frontend/Brave Notifications/Brave Wallet/WalletConnectionView.swift

```diff
@@ -12,15 +12,18 @@ class WalletConnectionView: UIControl {
   private let stackView: UIStackView = {
     let result = UIStackView()
     result.axis = .horizontal
-    result.spacing = 15
+    result.spacing = 20
     result.alignment = .center
+    result.distribution = .fill
```

Comment:
`.fill` is default distribution for the stack view, can be skipped

Reply:
addressed at [147fadf](https://github.com/brave/brave-ios/pull/5240/commits/147fadf2f3a3c8fde5c1d0a0d56bd55bb3f1f702)